frontend/coprs_frontend/coprs/logic/builds_logic.py

```diff
@@ -458,6 +458,13 @@ def create_new_from_other_build(cls, user, copr, source_build,
         build.pkg_version = source_build.pkg_version
         build.resubmitted_from_id = source_build.id
 
+        # Skip previously skipped chroots (ExcludeArch, ExclusiveArch, etc)
+        skipped = [ch.name for ch in source_build.build_chroots
+                   if ch.status == StatusEnum("skipped")]
+        for chroot in build.build_chroots:
+            if chroot.name in skipped:
+                chroot.status = StatusEnum("skipped")
```

Comment:
Couldn't we place this into backend_enqueue_buildchroots() and have the logic concentrated on one place?

Reply:
The `backend_enqueue_buildchroots` method is not called when resubmitting. Should I call it from this place in the code?

Reply:
I forgot how the resubmit works. It doesn't use the previous SRPM but rather builds it again. That means the whole python3-specfile parsing happens again and it is send to backend as `results.json` and then sent to frontend as for any other not-resubmitted build. Soooo, #2833 is invalid? When I run frontend from the `main` branch and resubmit package with exclusivearch, it skips the chroots.

Reply:
Yes, but  *resubnitted srpm ploads*. This is tricky...